app/views/characters/_editor.haml

```diff
@@ -22,34 +22,35 @@
         = label_tag :new_template, 'Create New Template'
   - if @character.user == current_user
     %tr#create_template{class: ('hidden' unless params[:new_template])}
-      %th.sub &#8627; Name
-      %td{class: klass}
-        = f.fields_for :template do |ff|
+      = f.fields_for :template do |ff|
+        %th.sub= ff.label :name, '↳ Name'
```

Comment:
What are the implications of using the character instead of the HTML &char equivalent?

Reply:
Either b/c it's in an equals or b/c it's singlequoted, that winds up rendering correctly and the &char doesn't seem to.

Reply:
It's not HTML-safe as a string, so Rails would re-escape it (`&amp;#8627;`). We could use the old thing with `.html_safe` on the end, but this seems fine.